README.md

```diff
@@ -114,11 +114,38 @@ shell alias to check your documentation changes build correctly.
 alias build-docs='RUSTDOCFLAGS="--cfg docsrs" cargo +nightly rustdoc --features="$FEATURES" -- -D rustdoc::broken-intra-doc-links'
 ```
 
-### Running benchmarks
+## Testing
+
+Unit and integration tests are available for those interested, along with benchmarks. For project
+developers, especially new contributors looking for something to work on, we do fuzz testing with
+[`Hongfuzz`](https://github.com/rust-fuzz/honggfuzz-rs), mutation testing with
+[`Mutagen`](https://github.com/rust-fuzz/honggfuzz-rs), and code verification with
```

Comment:
The mutagen link is wrong.

Also in raw text it reads lik Mutagen and Kani are fuzz testing since the links make right side harder to notice. Maybe separate the links?

Reply:
Thanks, changed to:
```
Unit and integration tests are available for those interested, along with benchmarks. For project
developers, especially new contributors looking for something to work on, we do:

- Fuzz testing with [`Hongfuzz`](https://github.com/rust-fuzz/honggfuzz-rs)
- Mutation testing with [`Mutagen`](https://github.com/llogiq/mutagen)
- Code verification with [`Kani`](https://github.com/model-checking/kani)

There are always more tests to write and more bugs to find, contributions to our testing efforts
extremely welcomed. Please consider testing code a first class citizen, we definitely do take PRs
improving and cleaning up test code.
```